config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
+  get 'me', to: 'dashboards#show'
   devise_for :users
   root to: 'pages#home'
+  resources :packages
+  resources :rentals, only: [:create, :update]
```

Comment:
does this need to be nested? would think that you would need a given package to create a new rental